src/main/java/duke/exception/IncorrectFormatException.java

```diff
@@ -0,0 +1,8 @@
+package duke.exception;
+
+public class IncorrectFormatException extends DukeException{
```

Comment:
I like how you created different subclasses of DukeException based on the type of user mistakes.